app/controllers/async_info_controller.rb

```diff
@@ -58,6 +58,9 @@ def user_data
         feed_style: feed_style_preference,
         created_at: @user.created_at,
         admin: @user.any_admin?,
+        policies: [
+          { resource_type: "Article", action: "create", allowed: policy(Article).create? },
```

Comment:
LOVE the `policy(Article).create?` I remember when you laid the groundwork for these kinds of questions. 😄